test.js

```diff
@@ -119,6 +122,53 @@ test('match and sort tags', function (t) {
   t.end()
 })
 
+test('parse tuples', function (t) {
+  t.same(parseTuple('linux-arm64'), {
+    name: 'linux-arm64',
+    platform: 'linux',
+    architectures: ['arm64']
+  })
+
+  t.same(parseTuple('darwin-x64_arm64'), {
```

Comment:
wdyt about using + instead _? darwin-x64+arm64

Reply:
I suppose `+` is less likely to cause issues, because there's prior art for arch names that contain `_`, like `x86_64`? SGTM

Reply:
👍